### PR TITLE
Move `/firefox/accounts` URL to `/account`

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
We've rebranded Firefox Accounts to Mozilla account back in late 2023, and we're now in the process of updating the URLs throughout Bedrock to match the visual and copy changes.

Check the issue to look at the required tasklist

## Significant changes and points to review
- Moved relevant files from the `/firefox/` directory to `/mozorg/`
- Updated `redirect.py` files to include redirects to the new URL
- Replaced links in Bedrock that link to `firefox.accounts` to updated `mozorg.account`
- Switched the layout in the `/account` page for better usability and readability (Jessica made the great suggestion)

> [!NOTE]
> - Many of the Fluent string IDs have `firefox` in them (i.e. `firefox-accounts-send-a-tab`), and there's no way to change that if we want to make sure our translations stay intact
> - The `firefox/accounts.ftl` file needs to remain in place so as to not break translations
> - This is a pretty big change, let me know if I missed anything?

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/14230

## Testing

- [ ] http://localhost:8000/en-US/firefox/accounts  👉🏼 redirected to 👉🏼 http://localhost:8000/en-US/account
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/edge/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/brave/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/safari/
- [ ] http://localhost:8000/en-US/firefox/features/password-manager/
- [ ] http://localhost:8000/en-US/firefox/features/sync/
- [ ] http://localhost:8000/en-US/firefox/features/tips/
- [ ] http://localhost:8000/en-US/firefox/new/
- [ ] http://localhost:8000/en-US/firefox/privacy/products/
- [ ] http://localhost:8000/en-US/firefox/set-as-default/thanks/ (I also updated the copy at the bottom, seems we missed updating it from `Firefox Accounts` during the overhaul in 2023)
- [ ] http://localhost:8000/en-US/firefox/welcome/6/
- [ ] http://localhost:8000/en-GB/firefox/122.0/whatsnew/
- [ ] http://localhost:8000/en-GB/firefox/ (bottom promo)
- [ ] http://localhost:8000/en-GB/products/ (bottom promo)
- [ ] http://localhost:8000/en-US/privacy/faq/

### CODE REVIEW

File: l10n/en/mozorg/account.ftl
Comment (alexgibson) [inline l10n/en/mozorg/account.ftl:1]: **note:** just a note that moving / renaming this file is going to break all existing translations, so we should probably resist changing it for now?

Comment (reemhamz) [reply l10n/en/mozorg/account.ftl:1]: D'oh! Good catch. This is one of the few key things about L10N I need to remember better.

Comment (reemhamz) [reply l10n/en/mozorg/account.ftl:1]: Is it just to refrain changing the file name, or also the location?

File: l10n/en/firefox/set-as-default/thanks.ftl [hunk 1]
```diff
@@ -27,9 +27,16 @@ set-as-default-thanks-youre-all-set = You’re all set.
...
 set-as-default-thanks-travel-the-internet-with = Travel the internet with protection on all your devices.
 set-as-default-thanks-download-the-app = Download the app
+
+# Obsolete string
 set-as-default-thanks-join-firefox = Join { -brand-name-firefox }
+
+# Obsolete string
 set-as-default-thanks-sign-up-for-a-free-account = Sign up for a free account and sync all your passwords, browsing history, and preferences across your devices.
 
+set-as-default-create-an-account = Create an account
```
Comment (reemhamz) [inline l10n/en/firefox/set-as-default/thanks.ftl:37]: I checked the brand guidelines and nowhere does it say to use "Join Mozilla" as a call to action for creating an account, so I instead went with this. Let me know if you think it should be named something else

Comment (stephaniehobson) [reply l10n/en/firefox/set-as-default/thanks.ftl:37]: The Join Mozilla branding is old. This is great.

**Review by alexgibson — APPROVED** [review]
This all looks great to me, nice thorough set of changes! r+

**Pull request merged**